build.sh

```diff
@@ -29,6 +29,31 @@ pushd "$ROOT_DIR/src/common/thirdparty/"
   bash build-redis.sh
 popd
 
+pushd "$ROOT_DIR/src"
+  rm -rf credis
+  git clone --recursive https://github.com/ray-project/credis
+popd
+
+pushd "$ROOT_DIR/src/credis"
+  git checkout 09120aa93791874428476ba780a27899753b484c
+
+  pushd redis && make -j && popd
```

Comment:
Is it possible to compile with tcmalloc by default?: `
pushd redis && env USE_TCMALLOC=yes make -j && popd`

At least, we should add a TODO here; tcmalloc bumps performance by a lot (and we should use it for current redis-server's as well).